docs/features/decorators.rst

```diff
@@ -0,0 +1,89 @@
+Decorators
+==========
+
+This section explains Robottello decorators.
+
+.. contents::
+
+Modules
+-------
+
+Robottelo decorators are located under
+:doc:`decorator package </api/robottelo.decorators>`. Most of then are used to
+control if a test must be skipped or executed accordingly with specific
+configurations.
+
+stubbed
+-------
+
+``stubbed`` skips any test it decorates. A reason can be provided as parameter
+and its default is "Not Implemented". Most of times it is used to define all
+test cases regarding a feature. Example::
```

Comment:
`Most of times it is used to define all manual(not automated) tests related to a feature` ?